modules/tinymce/src/themes/silver/main/ts/ui/skin/Loader.ts

```diff
@@ -53,15 +53,20 @@ const loadShadowDomUiSkins = async (editor: Editor, skinUrl: string): Promise<vo
 };
 
 const loadUrlSkin = async (isInline: boolean, editor: Editor): Promise<void> => {
-  Options.getSkinUrlOption(editor).fold(Fun.noop, (skinUrl) => {
+  Options.getSkinUrlOption(editor).fold(() => {
```

Comment:
Will this mean that content css will be loaded even if the skin url is not set? It used to be a noop?

Reply:
No it was not used to be a noop. I just forgot to add this part when the use does not specify a skin url. It is just the default behaviour.